CMakeLists.txt

```diff
@@ -82,6 +82,11 @@ option(GLM_TEST_ENABLE_SIMD_AVX "Enable AVX optimizations" OFF)
 option(GLM_TEST_ENABLE_SIMD_AVX2 "Enable AVX2 optimizations" OFF)
 option(GLM_TEST_FORCE_PURE "Force 'pure' instructions" OFF)
 
+option(GLM_ENABLE_EXPERIMENTAL "Enable experimental GLM_GTX" OFF)
+configure_file(glm/experimental.hpp.in experimental.hpp @ONLY)
```

Comment:
Why not simply: 
if GLM_ENABLE_EXPERIMENTAL 
add_definitions(-DGLM_ENABLE_EXPERIMENTAL)
?

Reply:
Does that make depending libs happy?

Reply:
Why not defining GLM_ENABLE_EXPERIMENTAL in the dependent library that includes GLM then?